chacra/async/__init__.py

```diff
@@ -0,0 +1,117 @@
+import pecan
+from celery import Celery
+import celery
+from datetime import timedelta
+from chacra import models
+from chacra.util import repo_directory
+import os
+import logging
+logger = logging.getLogger(__name__)
+
+
+def get_pecan_config():
+    try:
+        os.environ['PECAN_CONFIG']
+    except KeyError:
+        here = os.path.abspath(os.path.dirname(__file__))
+        config_path = os.path.abspath(os.path.join(here, '../config/config.py'))
+        return config_path
+
+
+pecan.configuration.set_config(get_pecan_config(), overwrite=True)
+# FIXME: this needs to be configurable. Also, not sure if this is the right way
+# to configure Celery for multiple tasks running in a single service.
```

Comment:
I think we're probably gonna have to use different celery queues and start multiple workers, one for each task.

http://celery.readthedocs.org/en/latest/userguide/routing.html